packages/headless/src/controllers/commerce/product-listing/summary/headless-product-listing-summary.ts

```diff
@@ -1,43 +1,3 @@
-import {CommerceEngine} from '../../../../app/commerce-engine/commerce-engine';
-import {
-  errorSelector,
-  isLoadingSelector,
-  numberOfProductsSelector,
-  responseIdSelector,
-} from '../../../../features/commerce/product-listing/product-listing-selectors';
-import {loadReducerError} from '../../../../utils/errors';
-import {
-  Summary,
-  SummaryState,
-  buildCoreSummary,
-} from '../../core/summary/headless-core-summary';
-import {loadProductListingReducer} from '../utils/load-product-listing-reducers';
+import {SummaryState} from '../../core/summary/headless-core-summary';
 
-export interface ListingSummaryState extends SummaryState {}
-
-export interface ListingSummary extends Summary {
-  /**
-   * A scoped and simplified part of the headless state that is relevant to the `ListingSummary` controller.
-   */
-  state: ListingSummaryState;
-}
-
-/**
- * Builds a `ListingSummary` controller.
- * @param engine
- * @returns
- */
-export const buildListingSummary = (engine: CommerceEngine) => {
-  if (!loadProductListingReducer(engine)) {
-    throw loadReducerError;
-  }
-
-  return buildCoreSummary(engine, {
-    options: {
-      selectNumProducts: numberOfProductsSelector,
-      selectResponseId: responseIdSelector,
-      selectLoading: isLoadingSelector,
-      selectError: errorSelector,
-    },
-  });
-};
+export interface ProductListingSummaryState extends SummaryState {}
```

Comment:
Eeeh feels a bit weird to have this in a file of its own, but I can't think of anything that would be better.